test/e2e/config/operator.yaml

```diff
@@ -35,8 +35,8 @@ variables:
   ARTIFACTS_FOLDER: "_artifacts"
   HELM_BINARY_PATH: "helm"
   HELM_EXTRA_VALUES_FOLDER: "/tmp"
-  KUBERNETES_VERSION: "v1.30.3"
-  KUBERNETES_MANAGEMENT_VERSION: "v1.30.0"
+  KUBERNETES_VERSION: "v1.30.4"
+  KUBERNETES_MANAGEMENT_VERSION: "v1.30.4"
```

Comment:
same, it is not out there yet: https://kubernetes.io/releases/? Kind image yes, v1.30.4 exists, but not k8s release as per the official k8s page

Reply:
Since we are using kind, it actually refers to kind image in our case.